pages/index.js

```diff
@@ -6,17 +6,20 @@ import SpeakersSection from '../components/SpeakersSection'
 import TicketsSection from '../components/TicketsSection'
 import AboutSection from '../components/AboutSection'
 import SponsorsSection from '../components/SponsorsSection'
+import LogoBackground from '../components/LogoBackground'
 
 export default () => (
   <MainLayout>
     <HeroUnit />
     <SectionSeparator first />
+    <LogoBackground width={820} top={100} right={-200} />
     <section id='about'><AboutSection /></section>
     <SectionSeparator />
     <section id='tickets'><TicketsSection /></section>
     <SectionSeparator />
     <section id='sponsors'><SponsorsSection /></section>
     <SectionSeparator />
+    <LogoBackground width={820} top={50} left={-200} />
```

Comment:
Since width is 820 in both cases, I think we can remove this prop and just put the `width` as a constant in `LogoBackground.js`.